src/main/java/com/example/todowithcouchbase/task/controller/TaskController.java

```diff
@@ -0,0 +1,30 @@
+package com.example.todowithcouchbase.task.controller;
+
+import com.example.todowithcouchbase.task.model.dto.request.SaveTaskRequest;
+import com.example.todowithcouchbase.task.model.dto.response.SaveTaskResponse;
+import com.example.todowithcouchbase.task.service.TaskService;
+import com.example.todowithcouchbase.common.model.dto.response.CustomResponse;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("api/v1/task")
+@Validated
+public class TaskController {
+
+    private final TaskService taskService;
+
```

Comment:
- Use this mapper 

```
private final TaskToTaskResponseMapper taskToTaskResponseMapper=
            TaskEntityToTaskResponseMapper.initialize();
```

Reply:
ı dont use this mapper due to the fact that in application process normally we dont use mapper in our controller so that I find suitable to use builder in this task

Reply:
In the controller, It should be used it.
Revise it. @artfulCoder98